velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -108,5 +108,17 @@ TEST_F(ProbabilityTest, normalCDF) {
       normal_cdf(0, kNan, 0.1985), "standardDeviation must be > 0");
 }
 
+TEST_F(ProbabilityTest, poissonCDF) {
+  const auto poissonCDF = [&](std::optional<double> lambda,
+                              std::optional<int64_t> value) {
+    return evaluateOnce<double>("poisson_cdf(c0, c1)", lambda, value);
+  };
+
+  EXPECT_EQ(0.91608205796869657, poissonCDF(3, 5));
```

Comment:
Please add tests for double max, min and infinity limits as in the above tests.

Reply:
@aditi-pandit Added tests